Automattic/jetpack
# Verbum Comments: Add verbum settings for simple sites

Fixes #

## Proposed changes:


Adds Verbum Comments settings to `/wp-admin/options-discussion.php` for Simple Sites.

`Verbum` - Toggle Verbum Comments experience
`Allow Blocks` - Toggle Verbum Comments blocks
`Greeting Text` - Change comments greeting text
`Color Scheme` - Update Verbum Comments color screen (transparent, light, dark) - defaults to transparent

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Checkout branch
* Sandbox public api and simple site.
* Build jetpack-mu-wpcom plugin and sync to sandbox (`pnpm run build-js && jetpack rsync mu-wpcom-plugin SANDBOX_PATH/ALIAS`)
* Navigate to `/wp-admin/options-discussion.php`, verify default values and verify that values are updating correctly on setting screen and on Verbum Comments for your simple site.
